man/systemd.network.xml

```diff
@@ -925,6 +925,21 @@ Table=1234</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>IPv4ProxyARPPrivateVLAN</varname></term>
+        <listitem>
+          <para>Takes a boolean. Configures proxy ARP private VLAN for IPv4, also known as VLAN aggregation,
+          private VLAN, source-port filtering, port-isolation or MAC-forced forwarding.
+
+          This variant of the ARP proxy technique will allow the ARP proxy to reply back to the same
+          interface.
+
+          See <ulink url="https://tools.ietf.org/html/rfc3069">RFC 3069</ulink>. When unset,
```

Comment:
note that docbook/xml will eat up all the empty lines you inserted here. if you want this to be three paragraphs you need to use <para>…</para> around each paragraph

Reply:
Gotcha!

Reply:
Done.

Reply:
I used 3 para, does that make sense?